pkg/controller/nodeipam/ipam/cloud_cidr_allocator.go

```diff
@@ -99,6 +113,22 @@ func NewCloudCIDRAllocator(client clientset.Interface, cloud cloudprovider.Inter
 		err := fmt.Errorf("cloudCIDRAllocator does not support %v provider", cloud.ProviderName())
 		return nil, err
 	}
+
+	var stackType clusterStackType
```

Comment:
it seems we have to default this to ipv4, otherwise this test fails https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/cloud-provider-gcp/652/pull-cloud-provider-gcp-scenario-kops-simple/1767587332136898560, I think is because of that

Reply:
Done